### DISCUSSION TITLE
Fresh installation with Tailwindcss shows error

### DISCUSSION THREAD

**Discussion by fixotech** [vercel/next.js#59526] [Help]
### Summary

With an untouched code and first build there´s an error: 

Unhandled Runtime Error
Error: chain is not set up
Call Stack
document.addEventListener.once
< anonymous > (1:6392)

What can this be?

Thanks!

**Comment by csvidit** [comment]
Could you add a link to the repo/code?

**Reply by fixotech** [reply]
The error has gone. Maybe the issue occurs due to low system memory, after restarting the system, 
I´ve tried a new installation and now it´s fine!

Thanks for your response!
